solidity/contracts/Rewards.sol

```diff
@@ -85,7 +87,8 @@ contract Rewards is Ownable {
     uint256 public totalRewards;
     // Rewards that haven't been allocated to finished intervals
     uint256 public unallocatedRewards;
-    // Rewards that have been paid out;
+    // Rewards that have been paid out from this contract - as a signer
+    // rewards or transferred to a new rewards contract.
     // `token.balanceOf(address(this))` should always equal
     // `totalRewards.sub(paidOutRewards)`
     uint256 public paidOutRewards;
```

Comment:
I think that since KEEPs can be transferred to a new contract now, it is not technically "paid out" which associates with giving KEEPs to an operator in this context. Maybe we should rename it to ex. `dispensedRewards` or `distributedRewards` or something along this way?

Reply:
I like it. I also had a feeling that something is wrong now with `paidOut*` part but couldn't find a better naming.